### PR TITLE
Fix -s argument

### LINKED ISSUE #7: The "-s" option is not working
Thank you for providing this useful Python package!

The `bib-string` command line option "-s" does not work as described in the README. It requires you to additionally also pass a bibliography file via "-f". I will provide a fix in a PR.

### PR DESCRIPTION
**Pull request by georg-wolflein**
Fixes #7.

### CODE REVIEW

File: notion_scholar/bibtex.py [hunk 1]
```diff
@@ -37,7 +37,7 @@ def get_bib_database_from_string(string: str) -> BibDatabase:
         string.
     """
     bibtex_parser = BibTexParser(
-        interpolate_strings=False,
+        # interpolate_strings=False,
```
Comment (thomashirtz) [inline notion_scholar/bibtex.py]: Why did you remove the string interpolation ?

**Review by thomashirtz — CHANGES_REQUESTED** [review]

File: notion_scholar/bibtex.py (hunk 1 above)
Comment (georg-wolflein) [reply notion_scholar/bibtex.py]: My bad, I forgot to undo that comment. Will update PR
